packages/commonwealth/tsconfig.json

```diff
@@ -20,11 +19,5 @@
     "./server",
     "./server.ts",
     "./scripts"
-  ],
-  "references": [
```

Comment:
this is how TS project references work, so we can build composite projects... not sure if there is a better way to achieve incremental builds though.